app/views/partials/_standalone.html.erb

```diff
@@ -1,5 +1,5 @@
 <section id="flow-standalone-pages" data-component="Expander">
-  <h2 class="govuk-heading-s"><%= t('pages.footer') %></h2>
+  <button class="govuk-heading-s"><%= t('pages.footer') %></button>
```

Comment:
This is not very good. Assuming a non-JS environment (yes, I know...) we can easily see the HTML is no longer semantic. H2 provides something that button does not.

We should look for an enhancement technique that neither duplicates the word output (so no H2 + button combo), and does not simply add a link within the H2 (yes, not illegal in HTML and sometimes used by others but really just a horrible cop-out solution).

Thoughts on a solution... (not saying any are correct or suitable): 
- Possibly, we can adjust the Expander element so that it does not completely hide the content (thinking something like we do with access-hidden text, maybe if the container area just had height of 1px rather than 0px, if that's the case). 
- Allowing things to be controlled by CSS so an Accessible user can easily override (e.g. without the Expander class or Expander.open - not sure if a thing - would be it is classed as visible). Someone with a screen reader (presumably the target audience for changes) might switch CSS off by default, for obvious reasons. Also, I think there might be some media query that relates to screen readers. Perhaps that is an option.
- Some kind of aria-something (haspopup, controls, expanded, role=button ??) solution that might help to work round issues. 
- Not closing the area by default so it is alway open for screen readers (TBH, I can't see there being much point in closing anyway).
- Maybe put aria-hidden on the H2 and use the class="hidden" technique for the button. Actually I don't like this option because it's still not semantic!
- Perhaps as per the above but the H2 doesn't have aria-hidden, so works as a heading, and the button is class="hidden' thing that says something suitable about revealing the related content.

Sorry, I'm out of ideas now.

Reply:
You're right. I hadn't thought this through properly. 

Well we're sort of both right. It should have an H2 for semantics, but it should also have the correct interactive elements for accessibility.

I think the proper structure for this is probably 'details/summary` but I'll investigate further as to what we can do.

Reply:
Have reverted this section of the PR, and will do the changes to the expander in a separate Pr as it's a bit larger in scope now.